<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>PC Nummer</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="spinBoxProductMin">
     <property name="maximum">
      <number>999</number>
     </property>
     <property name="value">
      <number>101</number>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>bis</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Verk√§ufer Nummer von</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="QSpinBox" name="spinBoxPc">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QSpinBox" name="spinBoxProductMax">
     <property name="maximum">
      <number>999</number>
     </property>
     <property name="value">
      <number>105</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="spinBoxSellerMin">
     <property name="maximum">
      <number>999</number>
     </property>
     <property name="value">
      <number>101</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Artikelnummer von</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>bis</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QSpinBox" name="spinBoxSellerMax">
     <property name="maximum">
      <number>999</number>
     </property>
     <property name="value">
      <number>102</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="5">
    <widget class="QPushButton" name="pushButtonOk">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
